app/scripts/move-popup/dimItemTag.directive.js

```diff
@@ -12,24 +12,22 @@
     `
   });
 
-  ItemTagController.$inject = ['$rootScope', 'SyncService', 'dimPlatformService', 'dimSettingsService', 'dimItemTagService'];
+  ItemTagController.$inject = ['$scope', '$rootScope', 'dimSettingsService'];
 
-  function ItemTagController($rootScope, SyncService, dimPlatformService, dimSettingsService, dimItemTagService) {
+  function ItemTagController($scope, $rootScope, dimSettingsService) {
     var vm = this;
 
     vm.settings = dimSettingsService;
-    vm.selected = _.find(vm.settings.itemTags, function(tag) {
-      return tag.type === vm.item.tag;
+    $scope.$watch('vm.item.dimInfo.tag', function() {
```

Comment:
I thought about that, or about using a `Proxy` to do the same thing. But I was worried about the lack of control to choose when to save.

The watcher is to update the selected property when the tag change, btw - not to save the tag.